waku.md

```diff
@@ -115,6 +115,7 @@ confirmations-enabled = BIT
 ; of a 64-bit floating point number. 
 ; Values of qNAN, sNAN, INF and -INF are not allowed.
 ; Negative values are also not allowed."
+pow             = 1*DIGIT "." 1*DIGIT
```

Comment:
What does it exactly mean? Only `0.0` is allowed?

Reply:
`1*` means 1 or more digits.

Reply:
I think it is OK, as the ABNF part does not specify how things should be encoded right? I mean, that's why we have the comments?

Reply:
@adambabik it simply is any numbers separated by a `.` followed by any numbers.